lib/core/datatable/components/datatable/datatable.component.ts

```diff
@@ -668,6 +679,10 @@ export class DataTableComponent implements AfterContentInit, OnChanges, DoCheck,
         return this.selectionMode && this.selectionMode.toLowerCase() === 'multiple';
     }
 
+    isFolder(row: DataRow): boolean {
```

Comment:
I think this is incorrect, the datatable component is a generic one, it should not have a notion of Folder.